applications/ExaquteSandboxApplication/custom_processes/divergence_process.cpp

```diff
@@ -0,0 +1,168 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/geometry_utilities.h"
+#include "utilities/variable_utils.h"
+#include "exaqute_sandbox_application_variables.h"
+
+// Application includes
+#include "custom_processes/divergence_process.h"
```

Comment:
Following @rubenzorrilla suggestions and for the sake of consistency I would also apply the convention to the filename:
divergence_process.cpp -> calculate_divergence_process.cpp
divergence_process.h -> calculate_divergence_process.h

Reply:
Correct, I forgot to change.